### PR TITLE
feat:[SRM-12820]: Fix canary and primary instances label in deploymen…

### PR DESCRIPTION
**Pull request by naveen-harness**
…t-activity-summary API.

## Checklist
- [x] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.
- [x] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/40069)

### CODE REVIEW

**Comment by naveen-harness** [comment]
trigger messagecheck

**Comment by n00bitax** [comment]
//300-cv-nextgen:[io.harness.cvng.analysis.services](https://app.harness.io/ng/io.harness.cvng.analysis.services).impl.VerificationJobInstanceAnalysisServiceImplTest FAILED in 3 out of 3 in 511.9s

**Comment by naveen-harness** [comment]
trigger codeformat

File: 300-cv-nextgen/src/test/java/io/harness/cvng/analysis/services/impl/VerificationJobInstanceAnalysisServiceImplTest.java [hunk 1]
```diff
@@ -404,8 +404,8 @@ public void testGetBlueGreenDeploymentAdditionalInfo_withoutAnalysesAndHostRecor
 
     assertThat(canaryBlueGreenAdditionalInfo).isNotNull();
     assertThat(canaryBlueGreenAdditionalInfo.getType()).isEqualTo(VerificationJobType.BLUE_GREEN);
-    assertThat(canaryBlueGreenAdditionalInfo.getPrimaryInstancesLabel()).isEqualTo("blue");
-    assertThat(canaryBlueGreenAdditionalInfo.getCanaryInstancesLabel()).isEqualTo("green");
+    assertThat(canaryBlueGreenAdditionalInfo.getPrimaryInstancesLabel()).isEqualTo("before");
+    assertThat(canaryBlueGreenAdditionalInfo.getCanaryInstancesLabel()).isEqualTo("after");
```
Comment (n00bitax) [inline 300-cv-nextgen/src/test/java/io/harness/cvng/analysis/services/impl/VerificationJobInstanceAnalysisServiceImplTest.java:408]: if this is a value used across multiple places, we should declare it and use that constant.

**Review by n00bitax — APPROVED** [review]

**Review by AbhijithHarness — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/40069)* status: :shipit: complete! ✓

**Pull request merged**